src/LiberisLabs.CompaniesHouse/Response/CompanyFiling/FilingHistoryItem.cs

```diff
@@ -29,6 +30,9 @@ public class FilingHistoryItem
         [JsonProperty(PropertyName = "description")]
         public string Description { get; set; }
 
+        [JsonProperty(PropertyName = "description_values")]
+        public Dictionary<string, dynamic> DescriptionValues { get; set; }
```

Comment:
What does this return back? This will end up passing back a `JObject` which isn't really useful for the consumer.